extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,310 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Check;
+use Email\Tracking\TrackingCategories;
+
+abstract class FounderDigestController extends EmailController {
+	// Defaults; will be overridden in subclasses
+	const TRACKING_CATEGORY_EN = TrackingCategories::DEFAULT_CATEGORY;
+	const TRACKING_CATEGORY_INT = TrackingCategories::DEFAULT_CATEGORY;
+
+	protected $wikiId;
+	protected $pageViews;
+
+	public function initEmail() {
+		$this->wikiId = $this->request->getVal( 'wikiId' );
+		$this->pageViews = $this->request->getVal( 'pageViews' );
+		$this->assertValidParams();
+
+		$this->pageViews = number_format( $this->pageViews );
+	}
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-founder-digest-summary',
+			\WikiMap::getWikiName( $this->wikiId ) )->parse();
```

Comment:
Thanks for the info. I don't know what I was thinking when I added this. I must have forgot that ```{{SITENAME}}``` can be -- and was -- used.